lib/models/src/test/java/org/sagebionetworks/repo/model/drs/DrsObjectIdTest.java

```diff
@@ -0,0 +1,88 @@
+package org.sagebionetworks.repo.model.drs;
+
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.repo.model.entity.IdAndVersion;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+public class DrsObjectIdTest {
+
+    private IdAndVersion idAndVersion = IdAndVersion.parse(ENTITY_OBJECT_ID);
+    private static final String ENTITY_OBJECT_ID = "syn54321.1";
+
+    @Test
+    public void testCreateDrsObjectIdWithFileHandleId() {
+        // call under test
+        DrsObjectId drsObjectId = DrsObjectId.parse("fh12345");
+        assertEquals(DrsObjectId.Type.FILE_HANDLE, drsObjectId.getType());
+        assertEquals("12345", drsObjectId.getFileHandleId().toString());
+    }
+
+    @Test
+    public void testCreateDrsObjectIdWithEntityId() {
+        // call under test
+        DrsObjectId drsObjectId = DrsObjectId.parse("syn54321.1");
+        assertEquals(DrsObjectId.Type.ENTITY, drsObjectId.getType());
+        assertEquals("54321", drsObjectId.getEntityId().getId().toString());
+    }
+
+    @Test
+    public void testCreateDrsObjectIdWithNull() {
+        // call under test
+        IllegalArgumentException exception = assertThrows(IllegalArgumentException.class, () ->
+                DrsObjectId.parse(null));
+        assertEquals("objectId is required.", exception.getMessage());
+    }
+
+    @Test
+    public void testGetEntityId() {
```

Comment:
We can do grouping of same test methods to get better understanding of tested scenarios